presto-native-execution/src/test/java/com/facebook/presto/nativeworker/AbstractTestNativeGeneralQueries.java

```diff
@@ -118,7 +118,8 @@ public void testCatalogWithCacheEnabled()
         }
     }
 
-    @Test
+    // disabling due to flakiness https://github.com/prestodb/presto/issues/21820
```

Comment:
Should we investigate and fix these failures instead of disabling? There are a lot of queries here being disabled? Maybe we could break down this test method into smaller ones?

Reply:
i split it up into 6 tests rather than disabling.